drivers/include/cc110x.h

```diff
@@ -240,6 +240,13 @@ extern "C" {
 #define CC110X_DEFAULT_PROTOCOL         (GNRC_NETTYPE_UNDEF)
 #endif
 
+#ifndef CC110X_DEFAULT_CHANNEL
+/**
+ * @brief The default channel to set up after initializing the device
+ */
+#define CC110X_DEFAULT_CHANNEL      (0U)
```

Comment:
Not `IEEE802154_DEFAULT_SUBGHZ_CHANNEL`?

Reply:
The CC1100/CC1101 is not an IEEE 802.15.4 device. Also, `CC110X_DEFAULT_CHANNEL` is still around from the old driver, so keeping the name will keep old code working. `IEEE802154_DEFAULT_SUBGHZ_CHANNEL` is also set to 5 by default in e.g. `examples/gnrc_networking`, which is only available to the 868 MHz version of the CC1101. (The 433MHz ISM band has less license free bandwidth, resulting in only channels 0-4 being provided then.)

Reply:
I thought more of
```
#define CC110X_DEFAULT_CHANNEL (IEEE802154_DEFAULT_SUBGHZ_CHANNEL)
```
like at86rf2xx does.

But you are right, if it's not 802.15.4 that would be confusing.
I'm not sure why RIOT always defaults to the highest channel in a band, but I suppose changing that would cause many unhappy users.